src/bootstrap/config.rs

```diff
@@ -430,7 +430,7 @@ impl Config {
             }
         }).unwrap_or_else(|| TomlConfig::default());
 
-        let build = toml.build.clone().unwrap_or(Build::default());
```

Comment:
I think here and in many places further down this could be replaced with lazy `unwrap_or_default()`, which may even further reduce verbosity (e.g. `unwrap_or_else(|| (String::new(), String::new()))`)

Reply:
Definitely; I'll update the commit soon.